graphviz/dot.py

```diff
@@ -67,6 +67,7 @@ def __init__(self, name=None, comment=None,
         self.body = list(body) if body is not None else []
 
         self.strict = strict
+        self.enconding = encoding
```

Comment:
This is handled by the `super()`-call in line 61 above:

https://github.com/xflr6/graphviz/blob/0c503a32f5f17f99a6e0ad8ac29fb441e3c0a9f2/graphviz/files.py#L103